src/lib/core/partialcubeop.test.ts

```diff
@@ -536,3 +536,358 @@ describe("3x3 half turn", () => {
 
   expectAllOtherFacesUnaffected(CUBE_FACE_3X3, partial.halfTurn);
 });
+
+const CUBE_M_LAYER_2X2 = {
```

Comment:
technical callout here, M doesn't exist for 2x2 (only exists for odd number of layers of course), are you just treating it as an L turn?